source/_integrations/template.markdown

```diff
@@ -211,6 +211,8 @@ template:
     description: Defines a template to get the name of the entity.
     required: false
     type: template
+  object_id:
+    description: Use `object_id` instead of name for automatic generation of the entity id.  When used without a `unique_id`, the entity id will update during restart or reload if the entity id is available.  If the entity id already exists, the entity id will be created with a number at the end. When used with a `unique_id`, the `object_id` is only used when the entity is added for the first time. When set, this overrides a user-customized Entity ID in case the entity was deleted and added again.
   unique_id:
     description: An ID that uniquely identifies this entity. Will be combined with the unique ID of the configuration block if available. This allows changing the `name`, `icon` and `entity_id` from the web interface.
```

Comment:
shouldnt the new object_id be mentioned here too? 

> This allows changing the

Reply:
Yes, I can add it here as well.  There are rumblings that the upstream PR may be reverted.